### DISCUSSION TITLE
记一个 Base64 引发的 Bug — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#235] [Announcements]
https://mazhuang.org/2020/03/01/base64-bug/#0x01-%E9%81%87%E5%88%B0%E7%9A%84%E7%8E%B0%E8%B1%A1 

 本文原计划写两部分内容，第一是记录最近遇到的与 Base64 有关的 Bug，第二是 Base64 编码的原理详解。结果写了一半发现，诶？不复杂的一个事儿怎么也要讲这么长？不利于阅读和理解啊（其实是今天有点懒想去休闲娱乐会儿），所以 Base64 编码的原理详解的部分将在下一篇带来，敬请关注。

**Comment by VatalityWang** [comment]
路过

**Comment by zxning** [comment]
赞一个
